### PR TITLE
Fix UG and DG

### PR DESCRIPTION
**Pull request by aexolate**
**Use case: UC9 - Import data file**

**MSS**

1. User requests to import the data file from a specified file path.
1. EmployeeManager reads the data file from the specified file path.
1. EmployeeManager shows the updated list of employees imported from the data file.
1. Use case ends.

**Extensions**

* 1a. The user enters an invalid file path.
    * 1a1. EmployeeManager shows an error message.
    * Use case ends.
* 1b. The user enters a file path where the file doesn't exist.
    * 1b1. EmployeeManager shows an error message.
    * Use case ends.
* 1c. The user enters a file path where the data file is corrupted.
    * 1c1. EmployeeManager shows an error message.
    * Use case ends.

I modified my use case to just show an error message and end for extensions. Would this be a good idea?

### CODE REVIEW

**Review by Weiennn — APPROVED** [review]
LGTM, yepp i think we should change the UC

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -635,49 +633,57 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
-**Use case: UC8 - Exporting the data file**
+**Use case: UC8 - Sort the displayed list**
+
+**MSS**
+
+1. User requests to sort the displayed list based on an attribute.
+1. EmployeeManager displays the sorted list based on the specified attribute.
+1. Use case ends.
+
+**Extensions**
```
Comment (Goh-Li-Ting) [inline docs/DeveloperGuide.md:644]: Seems to be missing the extension for multiple attributes inputted

File: docs/UserGuide.md [hunk 2]
```diff
@@ -352,7 +358,7 @@ Imports an EmployeeManager data file into the application.
 Format: `import FILE_PATH`
 
 * Refer to [Specifying file path](#specifying-file-path) for help on usage of `FILE_PATH`.
-* The data file must be of `.json` file type. Otherwise, an error will be thrown.
+* The data file must be of `.json` file type. Otherwise, an error will be shown.
```
Comment (Goh-Li-Ting) [inline docs/UserGuide.md:361]: Should it be "error message"?

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -644,10 +644,15 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 **Extensions**
 
 * 1a. The user enters an invalid attribute.
-    * 1a1. EmployeeManager informs user that the attribute is invalid.
-    * 1a2. User enters a new attribute.
-    * Steps 1a1-1a2 are repeated until the attribute is valid.
-    * Use case resumes from step 2.
+    * 1b1. EmployeeManager shows an error message.
```
Comment (Goh-Li-Ting) [inline docs/DeveloperGuide.md:647]: Should this be 1a1?

**Review by Goh-Li-Ting — APPROVED** [review]
LGTM

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -671,58 +670,62 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+**Extensions**
+
+* 1a. The user enters an invalid attribute.
+    * 1a1. EmployeeManager shows an error message.
+
+  Use case ends.
+
+
+* 1b. The user enters multiple attributes.
+    * 1b1. EmployeeManager shows an error message.
```
Comment (ricketytoc) [inline docs/DeveloperGuide.md:690]: Right now our application will sort based on the first attribute given right instead of showing an error message?

Comment (Goh-Li-Ting) [reply docs/DeveloperGuide.md:690]: It shows an error message for me

**Pull request merged**